rpcd/playbooks/stage-python-artifacts.yml

```diff
@@ -0,0 +1,184 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Stage the git and python artifacts
+  hosts: "{{ staging_host | default('localhost') }}"
+  user: root
+  vars:
+    upstream_artifact_base_url: "{{ rpco_mirror_base_url | default('https://rpc-repo.rackspace.com') }}"
+    http_proxy_env: "{{ lookup('env', 'http_proxy') | default('not_set', true) }}"
+    https_proxy_env: "{{ lookup('env', 'https_proxy') | default('not_set', true) }}"
+  tasks:
+
+    - name: Set the staging path
+      set_fact:
+        staging_path: |-
+          {%- if groups['repo_all'] is defined -%}
+          /openstack/{{ hostvars[groups['repo_all'][0]]['inventory_hostname'] }}/repo
+          {%- else -%}
+          /openstack/stage
+          {%- endif -%}
+      tags:
+        - always
+
+    - name: Fetch the upstream manifest file
+      uri:
+        url: "{{ upstream_artifact_base_url }}/os-releases/{{ rpc_release }}/MANIFEST.in"
+        return_content: yes
+        validate_certs: yes
+      register: manifest
+      tags:
+        - always
+
+    - name: Fetch the upstream venv list
```

Comment:
@odyssey4me I'd be happy to see @cloudnull patch backported to master.
If you are waiting for this, please mark this with a `#TODO(odyssey4me): Remove this when  https://review.openstack.org/#/c/433152/  is backported to N`